buildtools/dep/dep.go

```diff
@@ -13,6 +15,25 @@ import (
 
 // ErrNoLockfile is returned if a dep manifest is found without an accompanying lockfile.
 var ErrNoLockfile = errors.New("found Gopkg.toml but not Gopkg.lock")
+var ErrIsIgnored = errors.New("this is ignored")
+
+// A Resolver is a wrapper for Lockfile to allow for dep specific toml file behavior. Inherits resolvers
```

Comment:
[nit] This comment is a bit misleading. Both the lockfile and the manifest are dep-specific. Also, it _implements_ `Resolver`. Go has no notion of inheritance (from a technical perspective, it has no notion of subtyping).